dspace-api/src/main/java/org/dspace/app/util/Configuration.java

```diff
@@ -51,6 +51,8 @@ public static void main(String[] argv) {
                           "optional name of the module in which 'property' exists");
         options.addOption("r", "raw", false,
                           "do not do property substitution on the value");
+        options.addOption("a", "all", false,
```

Comment:
@mwoodiupui : Yes, I'd prefer we replace `--all` with `--first`.  While I understand that it could risk breaking scripts which call `dsprop`, I think that behavior is more logical.  I'm worried people will otherwise assume a property only has one value if they forget to pass the `--all` flag.